plugins/battery/SleepValues.qml

```diff
@@ -33,30 +33,24 @@ ItemPage {
     objectName: "sleepValues"
     flickable: scrollWidget
 
-    property alias usePowerd: batteryBackend.powerdRunning
+    readonly property alias usePowerd: batteryBackend.powerdRunning
     property bool lockOnSuspend
-    property variant idleValues: [30,60,120,180,240,300,600,0]
+    readonly property variant idleValues: [30, 60, 120, 180, 240, 300, 600, 0]
 
     UbuntuBatteryPanel {
         id: batteryBackend
     }
 
     GSettings {
         id: powerSettings
-        schema.id: usePowerd ? "com.ubuntu.touch.system" : "org.gnome.desktop.session"
+        schema.id: root.usePowerd ? "com.ubuntu.touch.system" : "org.gnome.desktop.session"
         onChanged: {
             if (key == "activityTimeout" || key == "idleDelay") {
-                var curIndex = idleValues.indexOf(value)
+                var curIndex = root.idleValues.indexOf(value)
                 if( curIndex != -1)
                     sleepSelector.selectedIndex = curIndex
```

Comment:
Since you are binding selectedIndex below, do we need to set the value here?

Reply:
Yes, because if system-settings is open to this page, and the setting is modified externally (such as via `gsettings` CLI tool), the UI should be updated to reflect that change.

Reply:
I mean you bind to powerSettings.activityTimeout (inside of a indexOf), wont that trigger a update to the binding if it changes?

Reply:
I would have to dig deep into gsettings-qt I guess to figure out for sure if it would or not, but this code was already here, and things are working as expected without removing it.

Reply:
make sense